.vscode/launch.json

```diff
@@ -96,37 +96,37 @@
             "type": "node-terminal",
         },
         {
-            "command": "cd packages/editor && npm run test",
+            "command": "cd packages/editor && npm run test -- --maxConcurrency 1 --no-file-parallelism --bail 1 --reporter=basic",
```

Comment:
why are we declaring separate parameters here? can we include these in the config fies?

Reply:
They are here to restore the behavior expected when debugging with vscode. Otherwise stepping through the code is very slow and results are unpredictable and non-reproducible due to concurrency, the code not stopping on errors, and the terminal output being overwritten very aggressively by `--reporter=default`

These options only make sense in the context of vscode debugging, that's why they are here
The goal eventually is to change this to `--project=spatial.vscode` and manage these options from the workspace config instead, but this is just to get by in the meantime